tests/fourier/test_reconstruct.py

```diff
@@ -918,6 +918,8 @@ def test_differentiability_jax(
         """Tests the reconstruction and differentiability with JAX."""
         import jax
 
+        jax.config.update("jax_enable_x64", True)
```

Comment:
It is really annoying to have to specify this every time :weary:  Why are these tests being modified here though?

Reply:
I just had an idea Ill see if it can work shortly.

Reply:
Nevermind. The tests keep breaking because the order of tests changes and tests that should have jax configured to run with `jax_enable_x64=True` are not. I am seeing if a fixture will fix this, but that should probably be its own PR.

Reply:
I just checked and adding ```@pytest.fixture(scope="function", autouse=True)
def update_jax_config():
    if jax_available:
        jax.config.update("jax_enable_x64", True)```

to `tests.conftests.py` should fix this.